From a0eadf8c7b3396d0c00679b40d2b7b790a92b11c Mon Sep 17 00:00:00 2001
From: Awais Belal <awais_belal@mentor.com>
Date: Mon, 17 Oct 2016 18:13:40 +0500
Subject: [PATCH] simpledecoder: fix build for OE

Signed-off-by: Awais Belal <awais_belal@mentor.com>
---
 CMakeLists.txt | 11 +++--------
 1 file changed, 3 insertions(+), 8 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b105abd..cce9848 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -80,6 +80,7 @@ else()
 find_path( VULKAN_INCLUDE_DIRS 
     NAMES vulkan.h
     HINTS ${VULKANROOT}/x86_64/include/vulkan/
+    HINTS /usr/include/vulkan/
 )
 
 find_library( VULKAN_LIBRARIES
@@ -88,12 +89,6 @@ find_library( VULKAN_LIBRARIES
     PATH_SUFFIXES ${PLATFORM}${BITNESS} ${BITNESS_SUFFIX}
 )
 mark_as_advanced( VULKAN_LIBRARIES )
-
-find_library( OPENH264_LIBRARIES
-    NAMES libopenh264.a 
-    HINTS ./lib/linux/ 
-    PATH_SUFFIXES ${PLATFORM}${BITNESS} ${BITNESS_SUFFIX}
-)
 endif()
 
 set( COMPILER_FLAGS " " )
@@ -120,7 +115,7 @@ if(CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX)
     
     set( COMPILER_FLAGS "${COMPILER_FLAGS} ${GCC_COVERAGE_COMPILER_FLAGS} ${EXTRA_COMPILER_FLAGS_GXX} " )
     set( LINKER_FLAGS "${LINKER_FLAGS} ${GCC_COVERAGE_COMPILER_FLAGS} ${EXTRA_LINKER_FLAGS_GXX} " )
-    set( ADDITIONAL_LIBRARIES ${ADDITIONAL_LIBRARIES} ${EXTRA_LIBRARIES_GXX} )
+    set( ADDITIONAL_LIBRARIES ${ADDITIONAL_LIBRARIES} ${EXTRA_LIBRARIES_GXX} ${CMAKE_CURRENT_SOURCE_DIR}/lib/linux/libopenh264.a )
 elseif( MSVC )
     # Samples can specify additional libs/flags using EXTRA* defines
     add_definitions( "/W3 /D_CRT_SECURE_NO_WARNINGS /wd4005 /wd4996 /nologo" )
@@ -146,7 +141,7 @@ endif( )
 if( MSVC )
     set(EXECUTABLE_OUTPUT_PATH ${CMAKE_CURRENT_SOURCE_DIR}/bin/${BITNESS_SUFFIX})
 else()
-    set(EXECUTABLE_OUTPUT_PATH ${CMAKE_CURRENT_SOURCE_DIR}/bin/${BITNESS_SUFFIX}/${CMAKE_BUILD_TYPE})
+    set(EXECUTABLE_OUTPUT_PATH ${CMAKE_INSTALL_BINDIR})
 endif()
 
 if( MSVC )
-- 
1.9.1

